kubetest/README.md

```diff
@@ -169,6 +170,44 @@ Tests can use `--skew` and `--upgrade_args` if they provided multiple
 as a sibling to `kubernetes/kubernetes`. This will cause tests to run from the
 skew directory, potentially to upgrade/downgrade kubernetes to another version.
 
+A simple example is:<br>
+`kubetest --up --check-version-skew=false --extract=v1.8.0-beta.1 
+--extract=v1.7.5 --upgrade_args=--ginkgo.focus="Feature:SomeUpgrade" 
+--gcp-project=google.com:some-project -v`
+
+The command runs all (and only) upgrade tests tagged with `Feature:SomeUpgrade` 
+label on GCE. The command downloads `v1.7.5` and `v1.8.0-beta.1` releases,
+unzips downloaded files, and runs the tests to upgrade the cluster from `v1.7.5` 
+to `v1.8.0-beta.1`. You will be able to find 2 new directories named `kubernetes`
+and `kubernetes_skew` at current directory. `kubernetes` is the directory
+corresponding to release indicated by first `--extract` flag, while `kubernetes_skew` 
+corresponds to second flag.
+
+Note that order of the 2 `extract` flags matters: `--extract=v2 --extract=v1` means
```

Comment:
bikeshed: s/2 `extract`/two `--extract`/

Reply:
huh?

Reply:
huh? do you mind to explain more about it?

Reply:
I mean add `--` before `extract` :-)

Reply:
got it! :)